pyca/capture.py

```diff
@@ -71,8 +71,14 @@ def start_capture(db, upcoming_event):
 
     # Set status
     # If part files exist, its an partial recording
-    p = any([glob.glob(f'{f}-part-*') for f in files])
-    state = Status.PARTIAL_RECORDING if p else Status.FINISHED_RECORDING
+    part_files = any([glob.glob(f'{f}-part-*') for f in files])
+    if part_files:
+        state = Status.PARTIAL_RECORDING
+    if config('agent')['backup_mode']:
```

Comment:
You always overwrite the status `PARTIAL_RECORDING`. If it's set. it should stay:

```suggestion
    elif config('agent', 'backup_mode'):
```

Reply:
Ah okay, i just copied that line from the old code, so the old code did not work anyway.^^